pyproject.toml

```diff
@@ -79,6 +79,7 @@ django-viewflow = "*"     # https://github.com/viewflow/viewflow
 
 # Dev dependencies (not installed on remote environments)
 unidecode = "^1.3.8"
+celery = "^5.4.0"
```

Comment:
Pas seulement une dev dependancy.
Juste un petit mismatch avec le commentaire "Dev dependencies"

Reply:
Je me demande pourquoi tu n'as pas eu besoin d'installer redis en dépendance ?
cf https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/redis.html#broker-redis
Pourtant redis est dans le requirement.txt

Reply:
idem, tu fix as déjà fixé mon 2° commentaire

Reply:
Tout à fait, fix ici : https://github.com/betagouv/complements-alimentaires/pull/720/commits/e029f883ff02a42ddeea5bc606f9f35e8f74606d